lib/TabTracker.js

```diff
@@ -33,30 +34,32 @@ TabTracker.prototype = {
   },
 
   getLastActivation: function(tab) {
-    if (!ss.storage.tabData[tab.url] || !ss.storage.tabData[tab.url][tab.id]) {
+    if (!this._tabData[tab.url] || !this._tabData[tab.url][tab.id]) {
```

Comment:
Since we will only ever have one active tab at any given time, it makes sense to make `tab.id` a property of `_tabData` as a sibling of `tab.url`.